eval/mt_bench/components.py

```diff
@@ -0,0 +1,171 @@
+# type: ignore
+# pylint: disable=no-value-for-parameter,import-outside-toplevel,import-error
+from typing import List, NamedTuple
+from kfp.dsl import component, Input, Output, Artifact, Model, importer
+from utils.consts import PYTHON_IMAGE
+
+EVAL_IMAGE = "quay.io/sallyom/instructlab-ocp:eval"
+
+@component(base_image=EVAL_IMAGE)
+def run_mt_bench_op(
+    models_path_prefix: str,
+    models_list: List[str],
+    mt_bench_output: Output[Artifact],
+    merge_system_user_message: bool,
+    max_workers: str = "auto",
+    device: str = None,
+) -> NamedTuple('outputs', best_model=str, best_score=float):
+
+    import subprocess
+    subprocess.run(["python3.11", "-m", "pip", "install", "vllm"], check=True)
+
```

Comment:
I think this is probably a better approach to get vllm into the component image. 
```suggestion
@component(base_image=EVAL_IMAGE, packages_to_install=["vllm"]))
def run_mt_bench_op(
    models_path_prefix: str,
    models_list: List[str],
    mt_bench_output: Output[Artifact],
    merge_system_user_message: bool,
    max_workers: str = "auto",
    device: str = None,
) -> NamedTuple('outputs', best_model=str, best_score=float):


```

Reply:
Can we force to use`@git+https://github.com/opendatahub-io/vllm@2024.08.01`?